plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/PlayerInfoCommand.java

```diff
@@ -44,8 +45,9 @@ public void sendInfoMessage(List<String> baseMessage, User infoUser, CommandSend
 
         for (String messageLine : baseMessage) {
             if (infoUser.hasGuild()) {
-                messageLine = StringUtils.replace(messageLine, "{GUILD}", infoUser.getGuild().getName());
-                messageLine = StringUtils.replace(messageLine, "{TAG}", infoUser.getGuild().getTag());
+                Guild guild = infoUser.getGuild().get();
+                messageLine = StringUtils.replace(messageLine, "{GUILD}", guild.getName());
+                messageLine = StringUtils.replace(messageLine, "{TAG}", guild.getTag());
             }
             else {
```

Comment:
user.getGuild.peek+onEmpty

Reply:
te messageLine wtedy musza byc effectiveFinal i ogólnie jest wiecej zabawy jakby to było zrobione peekiem